celery/backends/base.py

```diff
@@ -817,11 +817,25 @@ class BaseKeyValueStoreBackend(Backend):
     def __init__(self, *args, **kwargs):
         if hasattr(self.key_t, '__func__'):  # pragma: no cover
             self.key_t = self.key_t.__func__  # remove binding
-        self._encode_prefixes()
         super().__init__(*args, **kwargs)
+        self._add_global_keyprefix()
+        self._encode_prefixes()
         if self.implements_incr:
             self.apply_chord = self._apply_chord_incr
 
+    def _add_global_keyprefix(self):
```

Comment:
from my understanding, this feature needs to be documented properly

Reply:
@auvipy If you could elaborate on what you exactly mean here, that would help a lot on our side to adjust the PR as per your needs.

Anyways, the method already has docstrings and the feature is already documented based on my understanding: https://docs.celeryq.dev/en/master/search.html?q=keyprefix.